addons/auth_signup/data/mail_template_data.xml

```diff
@@ -341,5 +341,65 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Email template for new device login alert -->
+        <record id="alert_new_device_email" model="mail.template">
```

Comment:
:socks: if we stick to guidelines, this sould be more like "mail_template_auth_new_device_alert" (begin with model to know what is contains, then the module, then additional information)